app/tutorial_servo_control_can/src/TutorialServo.cpp

```diff
@@ -0,0 +1,17 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+ 
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees) : m_servoPwmOut(servoPin) {
+    m_servoRangeInDegrees = servoRangeInDegrees;
+    m_servoPwmOut.period(0.020); // Seconds
+}
+ 
+// What is this for?
```

Comment:
This is so that your code can be compatible with different types of servos, not just ones with 180 degrees of range.